install.js

```diff
@@ -79,16 +100,15 @@ const addScripts = packageRoot => {
       packageJson.scripts = {};
     }
 
-    // Cover your eyes, functional friends – mutation time!
-    if (!get(packageJson, "scripts.start", "").includes("operational-scripts")) {
-      packageJson.scripts.start = "operational-scripts start";
-    }
-    if (!get(packageJson, "scripts.build", "").includes("operational-scripts")) {
-      packageJson.scripts.build = "operational-scripts build";
-    }
-
-    packageJson.scripts.prepublishOnly = "operational-scripts prepare";
-    packageJson.scripts.test = "operational-scripts test";
+    replaceScripts(
```

Comment:
Shouldn't it be called `addScripts` since it now _doesn't_ replace?

Reply:
It still replaces I'd say, just on a condition.